openedx/core/djangoapps/user_api/views.py

```diff
@@ -279,6 +280,9 @@ def get(self, request):
                     required=self._is_field_required(field_name)
                 )
 
+        # Add any Enterprise forms if the app is enabled
```

Comment:
Are we adding forms or fields here?  Comment says forms, function leads me to believe it's fields?